### PR TITLE
feat: add login page

### LINKED ISSUE #130: 🚀 - create login page
### Description

create login page

### PR DESCRIPTION
**Pull request by MishaZhem**
Closes #130

### CODE REVIEW

**Comment by MishaZhem** [comment]
forgot password, login google

**Comment by MishaZhem** [comment]
forgot password, login by google..., register

File: apps/taiga-lumbermill/src/dashboards/login/login.component.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    ],
+})
+export class LoginComponent {
+    protected readonly form = new FormGroup({
+        email: new FormControl('', [Validators.required, Validators.email]),
+        password: new FormControl('', [Validators.required]),
+        rememberMe: new FormControl(false),
+    });
+
+    protected onSubmit(): void {}
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/login/login.component.ts:62]: let's add some interactivity on submit
loader button and redirect to main page

Comment (MishaZhem) [reply apps/taiga-lumbermill/src/dashboards/login/login.component.ts:62]: design

File: apps/taiga-lumbermill/src/dashboards/common-page/common-page.component.html [hunk 2]
```diff
@@ -49,4 +49,21 @@
...
+        tuiCardLarge="normal"
+        class="card"
+    >
+        <div>
+            <header tuiHeader>
+                <h2
+                    tuiTitle
+                    [style.text-align]="'center'"
+                >
+                    Login
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/common-page/common-page.component.html:64]: Log in. With space

File: apps/taiga-lumbermill/src/dashboards/login/login.component.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+            class="block-input"
+        >
+            password
+            <input tuiTextfieldLegacy />
+        </tui-input-password>
+        <tui-error
+            formControlName="password"
+            [error]="[] | tuiFieldError | async"
+            [style.margin-bottom]="'0.8rem'"
+            [style.margin-right]="'auto'"
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/login/login.component.html:46]: add css class for repeating styles

File: apps/taiga-lumbermill/src/dashboards/login/login.component.html [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+        <div class="under-password">
+            <label tuiLabel>
+                <input
+                    checked
+                    formControlName="rememberMe"
+                    tuiCheckbox
+                    type="checkbox"
+                />
+                <span
+                    tuiTitle
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/login/login.component.html:57]: this is definitely not a title

File: apps/taiga-lumbermill/src/dashboards/login/login.component.html [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+            [overlay]="true"
+            [showLoader]="submitLoader()"
+            [style.width]="'100%'"
+        >
+            <button
+                size="m"
+                tuiButton
+                [style.width]="'100%'"
+            >
+                Login
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/login/login.component.html:77]: Log in

File: apps/taiga-lumbermill/src/dashboards/common-page/common-page.component.html [hunk 6]
```diff
@@ -49,6 +49,23 @@
...
+        tuiCardLarge="normal"
+        class="card"
+    >
+        <div>
+            <header tuiHeader>
+                <h2
+                    tuiTitle
+                    [style.text-align]="'center'"
+                >
+                    Login in
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/common-page/common-page.component.html:64]: not login in, log in

File: apps/taiga-lumbermill/src/dashboards/login/login.component.html [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+    tuiAppearance="whiteblock"
+    tuiCardLarge="normal"
+    class="card"
+>
+    <header tuiHeader>
+        <h2
+            tuiTitle
+            [style.text-align]="'center'"
+        >
+            Login
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/login/login.component.html:11]: Log in

**Review by vladimirpotekhin — DISMISSED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**